lib/pbench/test/unit/server/query_apis/test_iteration_samples.py

```diff
@@ -679,3 +683,23 @@ def test_iteration_samples_with_scroll_id(
             assert response.json == {
                 "results": [hit["_source"] for hit in response_payload["hits"]["hits"]]
             }
+
+    def test_exceptions_on_get_index(self, attach_dataset):
+        test = Dataset.attach(controller="node", name="test")
+
+        # When server index_map is None we expect 500
+        with pytest.raises(InternalServerError) as exc:
+            self.cls_obj.get_index(test, "result-data-sample")
+        assert exc.value.code == HTTPStatus.INTERNAL_SERVER_ERROR
+
+        Metadata.setvalue(
+            dataset=test,
+            key="server.index-map",
```

Comment:
`Dataset.INDEX_MAP`??

Reply:
Ah, yeah if more changes are requested, I will fix this one as well.

Reply:
Fixed